command/sub8_missions/nodes/move_command

```diff
@@ -63,7 +63,7 @@ def main(args):
             last_digit_index = [i for i, c in enumerate(args.argument) if c.isdigit()][-1] + 1
             amount = args.argument[:last_digit_index]
             unit = args.argument[last_digit_index:]
-        amout = float(amount)
+        amount = float(amount)
```

Comment:
I think this line isn't even needed because of the cast to float in [line 70](https://github.com/uf-mil/SubjuGator/blob/master/command/sub8_missions/nodes/move_command#L70).